documentation/source/user_section/installation/windows.rst

```diff
@@ -343,53 +326,22 @@ First, save your Docker image:
-   - If this is the first time you have done this procedure, the system will ask you if you want to add the remote PC (the docker container) as trust pc, type yes. Then type the password to enter the docker container (by default sct).
-
-**Troubleshooting:**
-
-The graphic terminal emulator LXterminal should appear (if not check the task bar at the bottom of the screen), which allows copying and pasting commands, which makes it easier for users to use it. If there are no new open windows:
-
-- Please download and run the following file: :download:`Erase_fingerprint_docker.sh<../../../../contrib/docker/Erase_fingerprint_docker.sh>`
-- Try again
-- If it is still not working:
-
-  - Open the file manager and go to C:/Users/Your_username
-  - In the searchbar type ‘.ssh’ - Open the found ‘.ssh’ folder.
-  - Open the ‘known_hosts’ file with a text editor
-  - Remove line starting with ``192.168.99.100`` or ``localhost``
-  - Try again
-
-To check that X forwarding is working well write ``fsleyes &`` in LXterminal and FSLeyes should open, depending on how fast your computer is FSLeyes may take a few seconds to open. If fsleyes is not working in the LXterminal:
-
-- Check if it's working on the docker machine by running ``fsleyes &`` in the docker quickstart terminal
-- If it works, run all the commands in the docker terminal.
-- If it throws the error ``Unable to access the X Display, is $DISPLAY set properly?`` follow these next steps:
-
-  - Run ``echo $DISPLAY`` in the LXterminal
-  - Copy the output address
-  - Run ``export DISPLAY=<previously obtained address>`` in the docker quickstart terminal
-  - Run ``fsleyes &`` (in the docker quickstart terminal) to check if it is working. A new Xming window with fsleyes should appear.
-
-Notes:
-
-- If after closing a program with graphical interface (i.e. FSLeyes) LXterminal does not raise the shell ($) prompt then press Ctrl + C to finish closing the program.
-- Docker exposes the forwarded SSH server at different endpoints depending on whether Docker Desktop or Docker Toolbox is installed.
+      docker commit <CONTAINER_ID> <IMAGE_NAME>/ubuntu:ubuntu22.04
 
-  - Docker Desktop:
+4. Install `Xming <http://www.straightrunning.com/XmingNotes/>`_. The Public Domain version of Xming should be sufficient.
+   Alternatively, install `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_.
 
-    .. code:: bash
+5. Launch an X11 Server
 
-       ssh -Y -p 2222 sct@127.0.0.1
+- Run XLaunch, which should have been installed by default.
+- Check 'Multiple Windows' and take note of the **display number** > 'Start no Client' > 'No Access Control' > Finish
 
-  - Docker Toolbox:
+6. Determine the IP of the virtual Ethernet Adapter by running 'ipconfig' in Powershell or the Command Prompt.
 
-    .. code:: bash
+7. In your Terminal window, run:
+   
+   .. code:: bash 
+   
+      docker run -it --rm -e DISPLAY=<IP of vEthernet Adapter>:<Display Number> <IMAGE_NAME>/ubuntu:ubuntu22.04
 
```

Comment:
I was getting the same output of non-fatal warnings when I ran sct_check_dependencies.
Fortunately I was able to install VcXsrv to launch the X11 server instead of Xming, and the warnings went away!
The public Xming server version was released in 2007, and the VcXsrv version I downloaded was from 2022.

I have:
- linked to the VcXsrv download and provided instructions to set it up correctly
- added an XDG_RUNTIME_DIR environment variable to the docker run command
This should make a clean <sct_check_dependencies> reproducible.